README.md

```diff
@@ -197,8 +197,8 @@ When intra and inter op threads is set to 0 or a value higher than 1, by default
 #### Default Max Batch Size
 
 The default-max-batch-size value is used for max_batch_size during [Autocomplete](https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md#auto-generated-model-configuration) when no 
-other value is found. If the `--strict-model-config=false` command-line
-option is used, the onnxruntime backend will set the max_batch_size
+other value is found. Assuming server was not launched with`--disable-auto-complete-config`
```

Comment:
> Assuming server was not launched with`--disable-auto-complete-config`

Change to: "Assuming server was not launched with `--disable-auto-complete-config`" (adding white space between the back ticks)